tests/typescript/comments/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -702,9 +702,7 @@ interface Foo {
 }
 type T = </* comment */>(arg) => any;
 
-functionName<
-  A // comment
->();
+functionName<A>(); // comment
```

Comment:
**Prettier 2.1.2**
[Playground link](https://prettier.io/playground/#N4Igxg9gdgLgprEAuEAzArlMMCW0ByAhgLZwA8AOlAATUCC1A9I9ZMabBTFQHwAUASgDcIADQgIAB1zQAzslCEATkogB3AArKE8lIQA2awgE954gEZLCYANZwYAZUnWcUAObIYS9HHFxi5nAAJkHBADKE7uiEbnAAYhBKxIQwuO7IIIToMBBiIAAWMMT6AOr5OPCyzmBwDjoVOABuFcYZYLJmIK6ycEowGlZuycioBj3iAFayAB4AQla29g4kcGGucCNjviBT0w6ubvpwAIroEPCb+uMgzko9Shmo+up5kkquMCU4QTD5yAAcAAZxG8ID0SlZJBk3nB7o0NuIAI5neADKS6TKyAC0UDgwWCeSUcGROCJAxiwyQoyu2x6xBwnm8tIOR1O5w2VK24hghHMXx+fyQACZuVYcPoDgBhCDsQgZWEAVjy6B6ABVebpqddGj4AJJQUKwBxgd7SOgGhwwYxHS49AC+dqAA)
```sh
--parser flow
```

**Input:**
```jsx
functionName<
  A // comment	
>();
```

**Output:**
```jsx
functionName<A>(); // comment

```

**Prettier 2.1.2**
[Playground link](https://prettier.io/playground/#N4Igxg9gdgLgprEAuEAzArlMMCW0ByAhgLZwA8AOlAATUCC1A9I9ZMabBTFQHwAUASgDcIADQgIAB1zQAzslCEATkogB3AArKE8lIQA2awgE954gEZLCYANZwYAZUnWcUAObIYS9HHFxi5nAAJkHBADKE7uiEbnAAYhBKxIQwuO7IIIToMBBiIAAWMMT6AOr5OPCyzmBwDjoVOABuFcYZYLJmIK6ycEowGlZuycioBj3iAFayAB4AQla29g4kcGGucCNjviBT0w6ubvpwAIroEPCb+uMgzko9ShkwxpJwsmBKONJ5kh+wJThBGD5ZAADgADOIfhAeiUrJIMj9Xr1GhtxABHM7wAZSXSZWQAWigcGCwTySjgGJw5IGMWGSFGV22PWIOE83iZByOp3OG3pW3EMEI5n+gOBSAATAKrDh9AcAMIQdiEDKvACseXQPQAKkLdAzro0fABJKChWAOd6fGB0U0OJ5HS49AC+TqAA)
```sh
--parser typescript
```

**Input:**
```tsx
functionName<
  A // comment	
>();
```

**Output:**
```tsx
functionName<
  A // comment
>();

```

Reply:
I think the current TypeScript format is better. What do you think?

Reply:
Maybe, but I mean to consistent them with `flow`, let's improve it in another PR?

Reply:
Tracked #9537